index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>JSON Web Token</title>
+    <title>JSON Web Token Representation for Verifiable Credentials</title>
```

Comment:
Shouldn't it be something like `Securing Verifiable Credentials using JSON Web Tokens`?
Don't think we have agreement representations sit with the securing specs.

Reply:
`Representing VCs using JWTs and Securing VC-JWTs using JWS` might be a path forward, given that the full wording of each acronym will be in a nearby subtitle or introit...

Reply:
So I can merge / accept it?